searchcore/src/vespa/searchcore/proton/test/mock_document_db_reference.h

```diff
@@ -18,6 +18,9 @@ struct MockDocumentDBReference : public IDocumentDBReference {
     virtual std::shared_ptr<search::attribute::ReadableAttributeVector> getAttribute(vespalib::stringref) override {
         return std::shared_ptr<search::attribute::ReadableAttributeVector>();
     }
+    virtual std::shared_ptr<const search::IDocumentMetaStoreContext> getDocumentMetaStoreContext() const override {
```

Comment:
Consider renaming this to getDocumentMetaStore() to align with naming in ImportedAttributeVector